app/ClubEvent.php

```diff
@@ -45,7 +45,8 @@ class ClubEvent extends Model
 								'evnt_time_end',
 								'evnt_public_info',		
 								'evnt_private_details',
-								'evnt_is_private');
+								'evnt_is_private',
+                                'evnt_is_published');
```

Comment:
Some editors see tabs as two spaces, others as four, GitHub goes all nuts with 8. I've been haunted by the Spaces vs. Tabs dilemma. Use one or the other, just don't [mix](https://www.emacswiki.org/pics/static/TabsSpacesBoth.png) them :smiley: You might want to activate invisible characters in your IDE and make sure to use the formatter functionality to enforce some style guide.